sql/mob_skills.sql

```diff
@@ -3237,37 +3237,37 @@ INSERT INTO `mob_skills` VALUES (3408,2418,'eternal_misery',1,18.0,2000,1000,4,0
 INSERT INTO `mob_skills` VALUES (3409,2419,'crippling_agony',1,18.0,2000,1000,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (3410,2420,'bane_of_tartarus',1,18.0,2000,1000,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (3411,434,'ix_aern_mnk',0,7.0,2000,0,1,2,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (3412,436,'qn_aern_rdm',0,7.0,2000,0,1,2,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (3413,433,'qn_aern_whm',1,20.0,2000,0,1,2,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (3414,396,'blaster',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (3412,683,'freezebite',0,7.0,2000,0,1,2,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (3413,728,'combo',1,20.0,2000,0,1,2,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (3414,730,'one-ilm_punch',0,7.0,2000,1500,4,0,0,0,0,0,0);
```

Comment:
Did aern skills get moved?

Reply:
I guess they must have gone somewhere else, probably need to re-cap those aern skills

Reply:
I'm concerned about the mob skill lists which currently use these IDs:
3412 and 3413 are currently tied to skill list 3, Aern

3414 is tied to Sekhmet; I remember that @tankfest changed its ID for Charged Whisker back in #553 , so maybe _all_ of its IDs have changed.

@TeoTwawki have _skill IDs_ ever _changed_ before? This prospect scares me, because I don't think we have _any_ mechanism for dealing with _skill shifts_.

Reply:
@ibm2431 they have but its super rare - when it happenes it usually means the mobIDs also had a shift, some mob getting added or removed from zone in the middle of other mobs in the index.

Reply:
Those look like 2hrs on the Aerns, I suspect someone saw the blanks and assumed the aern 2hrs belong wshere when they really do not, and then later SE filled in the blanks with these.

Reply:
May want to ask @project-topaz/researcher to go let Sekhmet get some TP moves in so its existing skill list can be adjusted. If these skill IDs are getting replaced with those for trusts, we don't want to end up with a coeurl using Combo. 😉 

On the aern front, here are all the pools using that skill list ID:
![image](https://user-images.githubusercontent.com/13112942/87322615-242eed00-c51d-11ea-9574-6370b6613cc1.png)

Truth be told, I take aerns as being broken in general, and wouldn't mind simply removing those IDs from their lists. But if someone _really_ wants to hunt down what happened to those skills, there's a list of potential targets~

Reply:
re: sekhmet: https://github.com/project-topaz/topaz/pull/553#issuecomment-624654846

Reply:
iirc certain coeurl nm won't use whisker unless struck by magic 1st or it matches another condition like day. I think you have to thundaga sek or it needs lightning day before he'll use it. camped his drop on retail, only remember tripping it one time and then just avoided the trigger. don't remember what the trigger was exactly.

Reply:
I tried with multiple possibilities to make him use Charged Whisker, no success.

First time I pulled him with Frazzle (no damage), let him hit me for a good 15 minutes without casting anything else on him. It was during darksday through firesday, lasting more than an hour. Then I started casting Thunder on him until he went under 25% then under 20%. Still nothing.

I went back during lightningday then I tried to use Shock, Shock Spikes, Thundaga, Dia and multiple magic debuffs on him, even warping back because he was low on HP. Went back again and tried to melee him with low damage weapon. As far as being below 5% HP still during lightningday and... no trace of Charged Whisker. All of this lasted more than an hour too so he had time building up TPs hitting me, etc.

I check on YouTube, ZAM, BGWiki and FFXIclopedia. Beside FFXIclopedia and ZAM stating the same thing ("He rarely use it."), nothing can confirm it.

On a side note what can be confirmed about Sekhmet:

He is immune to Bind, Break, Gravity, Silence and Sleep (FFXIclopedia had Silence listed only). He doesn't link with other Coeurls and his Enaspir occurs at a really low rate (5-10% seems accurate).

Reply:
we should assume he has no skill at this point till someone proves he does (and how they got him to go)

I think its more likely everyone (incl me) mixed up their kitties than SE just spontaneously removing it.

And you forgot to tell them you tried to shock spikes to show kitty how to zap ppl. :wink: 

Conspiracy theory: SE accidentally gave poor this NM negative regain instead of positive.